scripts/Deepcell_Postprocessing.ipynb

```diff
@@ -165,26 +95,46 @@
    "outputs": [],
    "source": [
     "# load TIFFs of segmentation labels (from pixel clustering, deepcell, etc.)\n",
-    "segmentation_labels_preload = data_utils.tiffs_to_xr_labels(tiff_dir=label_dir, output_dir=None, delimiter='_')\n",
-    "plt.figure(figsize = (13, 13))\n",
-    "plt.imshow(segmentation_labels_preload.loc[\"Point8\", :, :, \"segmentation_label\"])"
+    "segmentation_labels = data_utils.tiffs_to_xr_labels(tiff_dir=label_dir, output_dir=None, delimiter='_feature_0')\n",
+    "if plot_labels:    \n",
+    "    for fov in segmentation_labels.fovs:\n",
+    "        plt.figure(figsize = (13, 13))\n",
+    "        plt.imshow(segmentation_labels.loc[fov, :, :, \"whole_cell\"])"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### We can also visualize the segmented mask overlaid on the imaging data"
+    "### We can also save and visualize the segmented mask overlaid on the imaging data"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "tags": []
+   },
    "outputs": [],
    "source": [
-    "plt.figure(figsize = (13, 13))\n",
-    "plt.imshow(plt.imread(os.path.join(segmentation_dir, \"Point8_HH3_Membrane_overlay.tiff\")))"
+    "\n",
+    "# get input data for overlay\n",
+    "input_data_xr = data_utils.load_imgs_from_multitiff([os.path.join(input_dir, point) for point in points_input])\n",
+    "\n",
+    "for fov in input_data_xr.fovs:\n",
+    "    plot_utils.plot_overlay(segmentation_labels.loc[fov, :, :, \"whole_cell\"].values,\n",
```

Comment:
Looks like some duplicated code in this cell. Let's keep the version that has the path specified so that the images get saved

Reply:
So, the code is dup'd because the path-specified version doesn't actually show the overlay, just saves it.  I'd pull up the saved images and show those, but some name/path reading logic is needed for that so I figured I'd go ahead and just run it again w/o saving to get the plots to show.

I could maybe just change `plot_utils.plot_overlay` to just show the plot anyways, or also take a bool to show plot or not

Reply:
I think given the number of different FOVs most people will have, it's fine to just save these as .tifs and let them inspect them one by one